addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -700,6 +700,12 @@ export function getOrCreateLink({ containerNode, startNode } = {}) {
         }
     } else if (!link && isContained) {
         link = document.createElement('a');
+        const commonAncestor = range.commonAncestorContainer;
+        const commonAncestorEl =  commonAncestor.nodeType !== Node.ELEMENT_NODE ?
+            commonAncestor.parentElement : commonAncestor;
+        if (commonAncestorEl.closest("p")) {
```

Comment:
This is our current assumption which adds inline translate class only inside of `p`. I think it deserves a comment to explain the reason